examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,14 @@
+''' A ac graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, defined start and
+end angles at defined point(coordinates).
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.arc
+    :refs: :ref:`ug_basic_lines_arcs
+    :keywords: arc, figure, glyph
+
+'''
```

Comment:
Also this change should belong to the other PR? Or do you intend to merge into a single PR? (either way typo on first line "A ac" -> "An arc")